cmd/gapic-showcase/run.go

```diff
@@ -60,6 +67,31 @@ func init() {
 				grpc.StreamInterceptor(observerRegistry.StreamInterceptor),
 				grpc.UnaryInterceptor(observerRegistry.UnaryInterceptor),
 			}
+
+			// load mutual TLS cert/key and root CA cert
+			if tlsCaCert != "" || tlsCert != "" || tlsKey != "" {
```

Comment:
Should this be `tlsCaCert != "" && tlsCert != "" && tlsKey != ""`? These variables all seem to be required in order to assemble the certificates.

Reply:
yes, updated the code.